### ISSUE TITLE
/bridgechain/ endpoint returns duplicate data 

### ISSUE THREAD

**Issue by Lemii** [ArkEcosystem/core#3122]
All but the very first bridgechain entry is (currently?) returned 6 times, as seen here:

https://dexplorer.ark.io/api/bridgechains

The offset query parameter also makes the endpoint behave unexpectetly, probably because of this. 

https://dexplorer.ark.io/api/bridgechains?offset=5

Furtermore, currently the totalCount does not match the actual returns objects because of the duplicates.

**Expected Behavior**

Display each bridgechain result in sequence and only once.

**Your Environment**
Devnet 2.6

**Comment by ArkEcosystemBot** [comment]
**Thanks for opening this issue!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by dated** [comment]
That is indeed the issue reported in #3111 😃 I submitted several PRs fixing the businesses and bridgechains API endpoint. The multiple indeces are however unfixed as of now.

**Comment by faustbrian** [comment]
Closing as duplicate of https://github.com/ArkEcosystem/core/issues/3111.

**Comment by ArkEcosystemBot** [comment]
This issue has been closed. If you wish to re-open it please provide additional information.
